pkg/common/label/label.go

```diff
@@ -0,0 +1,56 @@
+// Package label implements a standard set of Ginkgo labels for classifying
+// OpenShift Dedicated end to end platform/component tests.
+//
+// A label is for classifying or grouping a test based on what it is doing,
+// what is supports, and it's level of importance in decision making. These
+// labels can be used to target specific subsets of the overall suite during
+// execution by providing the Ginkgo/osde2e `--label-filter` flag.
+//
+// Within the `Describe` or `It` of a test, one to many labels can be included
+// to categorize the specific test(s) like so:
+//
+// var _ = Describe(testName, label.Blocking, func() { ... })
+//
+// Complex conditions can be provided to combine or negate different labels
+// such as `ROSA && Upgrade` or `e2e && !privatelink`. See the [Ginkgo
+// docs](https://onsi.github.io/ginkgo/#spec-labels) for more details. New
+// labels introduced should be generic and applicable to multiple test suites.
+//
+// Labels can also be used in conjunction with the `--focus-filter` flag to
+// provide the ability to run a specific suite's category of tests, for
+// example:
+// `--label-filter Install --focus-filter "Managed Cluster Validating Webhooks"`
+// `--label-filter Upgrade --focus-filter "Splunk Operator"`
```

Comment:
```suggestion
// Labels can also be used in conjunction with the `--focus` flag to
// provide the ability to run a specific suite's category of tests, for
// example:
// `--label-filter Install --focus "Managed Cluster Validating Webhooks"`
// `--label-filter Upgrade --focus "Splunk Operator"`
```

Suggestion: drop `filter` from the flag for `--focus`, to align with what comes from ginkgo directly.